modules/administration-guide/partials/assembly_authorizing-users.adoc

```diff
@@ -33,6 +33,4 @@ include::partial$proc_listing-che-permissions.adoc[leveloffset=+1]
 
 include::partial$proc_assigning-che-permissions.adoc[leveloffset=+1]
 
-include::partial$proc_sharing-che-permissions.adoc[leveloffset=+1]
```

Comment:
@themr0c This file does not seem to be included in **browse built doc**.
In any case, I opened a suggestion for this file in https://github.com/eclipse/che-docs/pull/2100/files